src/components/Label.js

```diff
@@ -9,7 +9,12 @@ const Label = ({labels}) => {
         return(
             <tr key={label.id}>
                 <td className="collapsing">
-                    <a className="ui basic label" style={{ backgroundColor:`#${label.color}` }}>{label.name}</a>
+                    <a className="ui basic label" style={{  font:'12px',
+                                                            backgroundColor:`#${label.color}`,
+                                                            opacity: '0.5',
+                                                            borderRadius:'2em',
+                                                            border:`solid grey 0.5px` 
+                                                            }}>{label.name}</a>
```

Comment:
```suggestion
                    <a className="ui basic label" style={{  font:'12px',
                        backgroundColor:`#${label.color}`,
                        opacity: '0.5',
                        borderRadius:'2em',
                         border:`solid grey 0.5px` 
                    }}>{label.name}</a>
```
Indention should be like this